plugins/delpoyment_stats/test/models/deployment_stats_test.rb

```
@@ -0,0 +1,48 @@
+require_relative '../test_helper'
```

rubocop (1.68.0)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)